### ISSUE TITLE
orcid logo

### ISSUE THREAD

**Issue by sje30** [ReScience/ReScience#88]
Would you like some latex code for rendering the ORCID logo in a slightly more orthodox fashion?

For our CODECHECK certificates (e.g. https://github.com/codecheckers/2020-cov-tracing/blob/master/codecheck/codecheck.pdf)
I used this code:
https://tex.stackexchange.com/questions/445563/ieeetran-how-to-include-orcid-in-tex-pdf-with-pdflatex/445583
see https://github.com/codecheckers/2020-cov-tracing/blob/master/codecheck/codecheck-preamble.sty

**Comment by rougier** [comment]
Oh nice! And yes, we could definitely use it, thanks. Can you make a PR for the latex template?

**Comment by sje30** [comment]
I can try (been struggling a bit with tex goodness).   But how would you lke the authors rendered, given you also use superscript for addresses, affiliations?  J Neuroscience does this, which I think is quite nice:

![2020-05-03_1680x1050](https://user-images.githubusercontent.com/1095067/80908532-821ea780-8d18-11ea-9ede-1343be4c79bd.png)

i.e logo then name then superscripts for author affiliations?

**Comment by rougier** [comment]
But they have a problem with spacing obviously and the logo is vertically misplaced. I will check on ORCID site if they have any recommendation.

**Comment by sje30** [comment]
Yes, the spacing is off at j neurosci.

I am happy with the spaced and slightly smaller version I made, see the SE link

On 3 May 2020 08:58:03 BST, "Nicolas P. Rougier" <notifications@github.com> wrote:
>But they have a problem with spacing obviously and the logo is
>vertically misplaced. I will check on ORCID site if they have any
>recommendation.
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/ReScience/ReScience/issues/88#issuecomment-623071095

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by rougier** [comment]
Yep, better. I'll try to adapt your code and we'll stick to the superscript version after affiliation.
